charts/csi-secrets-store-provider-aws/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- name: secrets-store-csi-driver
+  repository: https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
+  version: 0.0.23
```

Comment:
There have been a 0.1.0 release, while you are at it please jump to the latest release.

Reply:
Done